build.gradle.kts

```diff
@@ -21,7 +28,7 @@ kotlin {
     nodejs()
   }
 
-  linuxX64()
+  // linuxX64()
```

Comment:
This took a really long time on my machine. Is that expected? I commented it out to see if everything else would work (and it did).

Reply:
(this worked after stepping away from this PR for awhile)